scripts/encoding_pass.py

```diff
@@ -0,0 +1,27 @@
+from cryptography.fernet import Fernet
```

Comment:
Why Fernet? The most commonly used symmetric algorithm is the Advanced Encryption Standard (AES)